backend/backend/api/entries.py

```diff
@@ -202,10 +203,12 @@ def get(self, args, logbook_id=None):
             html = export_entries_as_html(logbook, entries)
             if html is None:
                 abort(400, message="Could not create HTML!")
-            return send_file(html, mimetype="text/html",
+            ret = send_file(html.name, mimetype="text/html",
                              as_attachment=True,
-                             attachment_filename=("{logbook.name}.html"
-                                                  .format(logbook=logbook)))
+                             attachment_filename=(slugify("{logbook.name}.html"
```

Comment:
Do you think this the slugifying is really necessary? Shouldn't UTF-8 be OK in filenames nowadays? I'm asking because it is another dependency and I always like to think twice before adding those.

Reply:
I was also not sure about that either.
If you think filenames with '`´" etc. are okay, than I can remove it, no problem. I have not tested this for now myself.

Reply:
I'll do some testing.